pages/create-lab-result.tsx

```diff
@@ -176,18 +178,20 @@ const CreateLabResult = () => {
               onSetSelectedCellsStatus={setSelectedCellsStatus}
               onGridError={setGridError}
             />
-            <div className="button-panel-wrapper">
-              <div className="button-panel">
-                <Button
-                  variant="contained"
-                  onClick={submit}
-                  color="primary"
-                  disabled={submitting}
-                  startIcon={submitting && <LoadingIcon style={{color: 'white'}} size={20} />}
-                >
-                  Začať test
-                </Button>
-              </div>
+
+            <div className="button-panel">
+              <IconButton onClick={() => createPdf(title, getGridContent(title, grid))}>
```

Comment:
use `title || 'grid'` as with empty title it's called 'file'

Reply:
This is done